src/test/rpc/Tx_test.cpp

```diff
@@ -674,22 +716,42 @@ class Tx_test : public beast::unit_test::suite
                     cmpPaymentTx(result.second.transaction(), tx);
                 }
 
-                if (ledger && !b)
-                {
-                    auto rawMeta = ledger->txRead(id).second;
-                    if (rawMeta)
-                    {
-                        auto txMeta = std::make_shared<TxMeta>(
-                            id, ledger->seq(), *rawMeta);
-
-                        cmpMeta(result.second.meta(), txMeta);
-                        cmpDeliveredAmount(
-                            result.second.meta(),
-                            result.second.transaction(),
-                            txMeta,
-                            tx);
-                    }
-                }
+                if (not ledger or b)
```

Comment:
First time I've seen the alternative logical operators in this codebase. I'm not opposed, it's got a nice Pythonic feel, but I'm wondering if this was deliberate or accidental.

Reply:
I'd prefer that we stick to what the rest of the code base does.

Reply:
Ha, this was deliberate. I wanted to see what people would say. I like the alternative logical operators better than the traditional logical operators. I think they are clearer. But, consistency matters, so I can change them back.

Reply:
I think these operators are actually not supported on windows